app/assets/stylesheets/application.scss

```diff
@@ -17,3 +17,7 @@
 body {
   background: linear-gradient(138deg, rgba(23,21,58,1) 0%, rgba(76,37,99,0.9150035014005602) 100%);
 }
+
+// html,body {
```

Comment:
commented this out as the gradient looked a little strange on login, cheers

Reply:
Legend, thank you.